src/App.tsx

```diff
@@ -15,36 +18,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  ALPHABETICALLY = 'ALPHABETICALLY',
+  BY_LENGTH = 'BY_LENGTH',
+  DEFAULT = 'DEFAULT',
+}
+
+//export type RereversType = true | false;
+
+interface SortParams {
+  sortType: SortType;
+  isReverse: boolean;
+}
+
+function getPreparedGoods(goods: Good[], { sortType, isReverse }: SortParams) {
+  const preparedGoods = [...goods];
+
+  preparedGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case 'ALPHABETICALLY':
+        return good1.localeCompare(good2);
+
+      case 'BY_LENGTH':
```

Comment:
Use a enum here instead of strings
```suggestion
      case SortType. ALPHABETICALLY:
        return good1.localeCompare(good2);

      case SortType. BY_LENGTH:
```